storage/postgres/where_clause_tree.go

```diff
@@ -100,7 +102,10 @@ func criterionSQL(c query.Criterion, dbTags []tagType, tableAlias string) (strin
 	ttype := findTagType(dbTags, c.LeftOp)
 	dbCast := determineCastByType(ttype)
 	var clause string
-	if tableAlias != "" {
+	if c.Type == query.ExistQuery {
+		rightResolvedValue, _ := tsprintf(rightOpQueryValue.(string), c.TemplateParameters)
```

Comment:
the error should probably not be ignored

Reply:
Since now the tsprintf is done outside the Criterion, this err is no longer relevant here :)
Thanks for this, now the solution is more simple 👍